src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -40,6 +40,47 @@ public void execute_deleteValidName_success() {
         assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_deleteValidNameUnfilteredList_success() {
```

Comment:
Just a clarification, this tests for the fact that you can delete a person not currently displayed by name?

Reply:
Yup!